spec/lib/bulk_email_helper_spec.rb

```diff
@@ -219,23 +219,18 @@ class BulkEmailTest
   # Oracle blows up if you do a WHERE IN (...) clause with more than a 1000 items
   # so let's test it.
   # commented out because the creation of users takes so long. run it every once in a while
-  # context 'being ready for Oracle' do
-  #   before :each do
-  #     puts "creating users"
-  #     1001.times do
-  #       user = FactoryGirl.create(:user)
-  #       od = place_product_order(user, @facility, @product, @account)
-  #     end
-  #     puts 'users created'
-  #     OrderDetail.all.size.should == 1001
-  #     puts 'ensured order detail size'
-  #   end
-  #   it "should return 1001 users" do
-  #     puts 'doing query'
-  #     users = @controller.do_search(@params)
-  #     puts "type: #{users.class}"
-  #     users.size.should == 1001
-  #   end
-  # end
+  context "being ready for Oracle" do
+    before :each do
+      1001.times do
+        user = FactoryGirl.create(:user)
+        od = place_product_order(user, @facility, @product, @account)
+      end
+      expect(OrderDetail.all.size).to eq(1001)
+    end
+    it "should return 1001 users" do
```

Comment:
If this were going to hang around, there's some more minor cleanup that should happen ("should", `@` variables, spacing).